docs/hardwarelabels.md

```diff
@@ -6,9 +6,12 @@ title: ''
 import Registration from "!!raw-loader!@site/examples/quickstart/registration-hardware.yaml"
 
 ## Hardware Labels
+
 When a node is registered, hardware data is collected and made available to the MachineRegistration in the same way as [SMBIOS data](smbios.md).
 
-This data can be used for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md).
+This data can be used for easy identification and selection via a [MachineSelector](machineinventoryselectortemplate-reference.md).  
+
+Note that Elemental Teal nodes will run `elemental-register` at each boot and every 24 hours after that, so that labels will eventually be updated to new values.  
```

Comment:
That effectively means that (label) changes in `MachineRegistration` are re-synced to `MachineInventory` ?!

Reply:
This is already the case actually, it should be documented on the MachineRegistration side.
https://elemental.docs.rancher.com/next/machineregistration-reference#machineinventorylabels